### DISCUSSION TITLE
Find the author

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#725] [Galaxy Trivia] [answered]
> 💃 like no one is 👀,
❤️ like you’ll never be 🤕
🎤 like no one is 👂
🤗 like it’s heaven on 🌍.

**Comment by SANKARAMDAS** [comment]
The following words are attributed to a variety of people including Mark Twain, Satchel Paige, and William Purkey.

Sing like no one’s listening, love like you’ve never been hurt, dance like nobody’s watching, and live like it’s heaven on earth.

**Comment by peterbaccaro** [comment, marked as answer]
The author of the quote "Dance like nobody's watching; love like you've never been hurt. Sing like nobody's listening; live like it's heaven on earth" is often attributed to the American writer and educator, **William W. Purkey**. However, it is important to note that the origin of the quote is disputed, and its true authorship is uncertain.
